sdk/pim-eligibility-migration/eligibility_sync.py

```
@@ -0,0 +1,524 @@
+                result = update_policy_eligibility_mapping(prov_id, existing_id, body)
+                merged_by_fp[fp] = result
+                updated += 1
+                reason = eligibility_put_change_summary(current, body)
+                logger.info(
+                    "Updated eligibility %d/%d: %s (%s)",
+                    i + 1,
+                    len(desired_list),
+                    name,
+                    reason,
+                )
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)